### PR TITLE
XW-1618 | Static global nav on flexgrid

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/XW-1618

Ping @Wikia/x-wing

### CODE REVIEW

File: components/_global-navigation.scss [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+			height: 15px;
+			width: 42px;
+		}
+
+		&__powered-by {
+			padding: 2px 5px 0;
+		}
+	}
+
+	&__middle-column {
```
Comment (rogatty) [inline components/_global-navigation.scss]:
```
text-align: center
vertical-align: middle
```
I'd use `&__center-column`

File: dist/styles.scss [hunk 2]
```diff
@@ -1,2 +1,3 @@
 @import '../base/index';
 @import '../components/index';
+@import '../bower_components/flexboxgrid/dist/flexboxgrid.min.css';
```
Comment (rogatty) [inline dist/styles.scss]: It's already imported in `components/index`. Does it make sense to do it again?

File: components/_global-navigation.scss (hunk 1 above)
Comment (kvas-damian) [inline components/_global-navigation.scss]: Discussed offline - middle makes sense here ;)

File: base/index.scss [hunk 3]
```diff
@@ -1,3 +1,4 @@
+@import '../bower_components/flexboxgrid/dist/flexboxgrid';
```
Comment (tomasznapieralski) [inline base/index.scss]: i would move this import to `dist/styles.scss`, because `index` should contain partials in this directory not from outside

**Comment by tomasznapieralski** [comment]
One minor, LGTM

**Pull request merged**